### PR TITLE
Fix the stlink binary path

### PR DESCRIPTION
**Pull request by polaroi8d**
Fix the Stlink path for the binary file. Add to checkout for a single commit hash, and we don't need to update the README all the time when something is changes.

JerryScript-DCO-1.0-Signed-off-by: Levente Orban orbanl@inf.u-szeged.hu

### CODE REVIEW

File: targets/nuttx-stm32f4/README.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ git clone https://github.com/jerryscript-project/jerryscript.git
 git clone https://bitbucket.org/nuttx/nuttx.git
 git clone https://bitbucket.org/nuttx/apps.git
 git clone https://github.com/texane/stlink.git
+cd stlink
+git checkout 3d24377c2878fc328fb09b67cdfbda529d209417
```
Comment (akosthekiss) [inline targets/nuttx-stm32f4/README.md]: I'm not sure I agree with this. Why do we have to fix stlink to a given hash? In the commit message you've mentioned that this saved us from the need for frequent updates to the readme. But what is changing so frequently in stlink that triggers updates here?

If we specify the hash here, we will have a "last known good version", for sure. But we lose any potential benefit from the development that happens on stlink master. And if we wanted to follow up on that then we had to update readme with new "last known good version" hashes over and over again.

Comment (polaroi8d) [inline targets/nuttx-stm32f4/README.md]: @akosthekiss Okey, I agree with that. What do you want, just update the changed path for the binary, and that's all?

Comment (akosthekiss) [inline targets/nuttx-stm32f4/README.md]: I indeed _suggest_ to limit the changes to section 5. The file path to the stlink  binary is one thing. The other is the flashing instructions URL, where I'd suggest to link to the repository only, without the `#build-from-sources` (which does not exist).

Comment (polaroi8d) [inline targets/nuttx-stm32f4/README.md]: Updated following your opinion.

File: targets/nuttx-stm32f4/README.md [hunk 2]
```diff
@@ -85,12 +85,12 @@ make
 
 Connect Mini-USB for power supply and connect Micro-USB for `NSH` console.
 
-To configure `stlink` utility for flashing, follow the instructions [here](https://github.com/texane/stlink#build-from-sources).
+To configure `stlink` utility for flashing, follow the instructions in the official Stlink repository [here](https://github.com/texane/stlink).
```
Comment (akosthekiss) [inline targets/nuttx-stm32f4/README.md]: Consider this nitpicking, but why not put the link on "Stlink repository", why is "here" needed?

**Pull request merged**